region/tourian/main/Mother Brain Room.json

```diff
@@ -466,19 +563,30 @@
       ]
     },
     {
-      "link": [4, 1],
-      "name": "Base",
+      "link": [3, 8],
+      "name": "Mother Brain Ice Zebetite Skip (Out)",
+      "notable": true,
       "requires": [
-        "f_DefeatedMotherBrain"
+        "canTrickyUseFrozenEnemies"
```

Comment:
I couldn't get reverse ice skip or reverse remorph skip to work.  Thought it was possible, but I guess there is a trick to it 🤷